exawind/Pazy_Wing/README.md

```diff
@@ -0,0 +1,177 @@
+- Nalu-Wind:  &ensp;&ensp;&nbsp;&nbsp;&nbsp;  [``b9e4ae654b646ecd0501dd6391dc7537239c82db``](https://github.com/Exawind/nalu-wind/commit/b9e4ae654b646ecd0501dd6391dc7537239c82db)
+- AMR-Wind:   &ensp;&emsp;&nbsp; [``b61c01895e8eab388e4a3fd129e1db0f4fd0f534``]( https://github.com/Exawind/amr-wind/commit/b61c01895e8eab388e4a3fd129e1db0f4fd0f534) 
+- OpenFAST: &ensp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[``024dbc1816ca8caeefcc720b1099397730b1ec0a``](https://github.com/Exawind/amr-wind/commit/024dbc1816ca8caeefcc720b1099397730b1ec0a)
+
+## Postprocessing
+
+The primary QoI is wing tip deflection, which is recorded in the OpenFAST output file `pazy.out`.  The `post.sh` script will duplicate this output file, remove its header, and run the `post.py` python script.  This will yield a plot of the tip deflection history, together with an average value for the tip deflection.
+
+The files need to reproduce the plots in this document may be found in the `plotting` directory.  For the static deflection (low speed) cases, the final tip deflection (from the `post.py` script mentioned above) may be assembled for several windspeeds and angles of attack into the files `nalu3.dat`, `nalu5.dat`, and `nalu7.dat`.  The quantity in the first column is the wind speed, and the quantity in the second column is the tip deflection (in meters).  Running `python3 plot3.py` will reproduce the AoA=3 figure below, and the other figures may be obtained likewise.
+
+The `plotting` directory also contains the files needed to reproduce the flutter boundary figure.  For this, you will need to do a parameter sweep (the `setup.sh` script will help with this), and run cases for several windspeeds near the flutter onset speed for each AoA (3, 5, and 7 degrees).  From the deflection history (obtained from the `post.py` script), one can determine whether each case meets the criteria for "stable", "unstable", or "flutter", as discussed in the results section below.  These speeds can be added to the files `nalustable.dat`, `naluunstable.dat`, and `naluflut.dat`.  Then, the figure may be reproduced by running `python3 plotflut.py`.
+
+## Code Performance
+
+For these simulations, 90% of the cores were allocated to Nalu-Wind, and the remainder to AMR-Wind.  However, the timings indicated that adding yet more cores to Nalu-Wind would yield a speed improvement in this case.  Future studies are planned to determine the optimal resource balance.
+
+| Property              | Value                  |
+| :---                  | ---                    |
+| Wall-Clock Time       | 48 hours               |
+| Simulated Time        | 2.09 seconds           |
+| Number of Timesteps   | 34,380                 |
+| Number of CPUs        | 560                    |
+| Number of HPC Nodes   | 5                      |
+| Nalu Cells per CPU    | ~15,600                |
+
```

Comment:
We could add a clarifying note here about how we ran for 48 hours because that was the length of the allowed time on the queue, and not because it was the amount of time it took to simulation 2.09 seconds.  Also perhaps that 2.09 seconds was sufficient to get the results mentioned below.

Reply:
Done!